template.yaml

```diff
@@ -347,6 +337,19 @@ Resources:
               - lambda:InvokeFunction
             Resource: "*"
 
+  ScopusBucketsAccessPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: !Sub DefaultLambdaPermissions-${AWS::StackName}
+      Roles:
+        - !Ref ReadFromScopusBucketsRole
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - 's3:*'
```

Comment:
`- s3:GetObject` is sufficient for NvaScopusFunction I believe

Reply:
I will restrict the access after we have established that the handler works.